File: back/back/back/Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace back.Models;

public partial class Product
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    [Key]
    public long? Id { get; set; }

    public string Code { get; set; } = null!;

    public string Name { get; set; } = null!;

    public string Description { get; set; } = null!;

    public string? Image { get; set; }

    public decimal Price { get; set; }

    public string Category { get; set; } = null!;

    public decimal Quantity { get; set; }

    public string InventoryStatus { get; set; } = null!;

    public decimal? Rating { get; set; }
}

File: back java/Product-trial-feat/src/main/java/com/example/Product_trial_feat/model/Product.java
package com.example.Product_trial_feat.model;
import jakarta.persistence.*;
import jdk.jfr.Name;
import lombok.Data;
import lombok.Generated;


@Data
@Entity
@Table(name="product")
public class Product {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)

    private long id;
    private String code;
    private String name;
    private String description;
    private String image;
    private Integer price;
    private String category;
    private Integer quantity;
    private String inventoryStatus;
    private Integer rating;
}
